qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -931,7 +931,17 @@ def parameters_table(
                 value_dict["date_time"] = value_dict["date_time"].strftime("%Y-%m-%d %H:%M:%S.%f%z")
                 data.append(value_dict)
 
-        return data
+        columns = [
+            "group",
+            "schedule",
+            "parameter",
+            "qubits",
+            "value",
+            "valid",
+            "date_time",
+            "exp_id",
```

Comment:
```suggestion
            "parameter",
            "qubits",
            "schedule",
            "value",
            "group",
            "valid",
            "date_time",
            "exp_id",
```
Let's keep the same ordering of `parameter, qubit, schedule` as in the function `get_parameter_value` and other locations in the code. The `group` attribute is not as important as those three and should go further back in the list.